index.js

```diff
@@ -100,6 +165,8 @@ if (require.main === module) {
   main(process.version)
 } else {
   module.exports = {
-    isNodeVulnerable
+    isNodeVulnerable,
+    isNodeEOL,
+    isNodeSupportedMajor
```

Comment:
```suggestion
```
Considering we're calling all these functions inside `isNodeVulnerable` we don't need to export it.

Reply:
How to test it separately then ?